x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension/endpoint_policy_create_extension.tsx

```diff
@@ -217,7 +196,7 @@ export const EndpointPolicyCreateExtension = memo<PackagePolicyCreateExtensionCo
                   <strong>
                     <FormattedMessage
                       id="xpack.securitySolution.endpoint.ingestManager.createPackagePolicy.environments"
-                      defaultMessage="protect your tranditional endpoints or dynamic clound environments"
+                      defaultMessage="protect your transitional endpoints or dynamic cloud environments"
```

Comment:
or is this supposed to say `traditional`?

Reply:
ouch! great catch